roles/edpm_bootstrap/defaults/main.yml

```diff
@@ -31,6 +31,8 @@ edpm_bootstrap_packages_bootstrap:
   - crudini
   - jq
   - nftables
+  - nvme-cli
+  - openvswitch
```

Comment:
I'm wondering if this is a rebase artifact

Reply:
I assume so. I rebased on origin/main, got a conflict, and it was due to these lines. The pushed update fixed the conflict.

Reply:
ovs has been moved to its own role so you should drop it form here

Reply:
one other thing we shoudl not install this by default on all hosts.

not all hosts will have vms so we should only install nvme-cli on host that need it.

edpm nodes are not 1:1 mapping with compute nodes

so you should probably install this in a different role.

i also am not sure why the nvme-cli package is required on the host?
os-brick runs in the nova_compute container.

is there a package that just provides the kernel modules that you can use instead?

Reply:
> one other thing we shoudl not install this by default on all hosts.
> 
> not all hosts will have vms so we should only install nvme-cli on host that need it.
> 
> edpm nodes are not 1:1 mapping with compute nodes
> 
> so you should probably install this in a different role.

Hmm, OK. Where is the host-to-role mapping maintained? I need to know how to get the role included on the appropriate host.

> i also am not sure why the nvme-cli package is required on the host? os-brick runs in the nova_compute container.
> 
> is there a package that just provides the kernel modules that you can use instead?

I tried to explain that in the commit message. The nvme-cli package initializes several files and udev rules.

Reply:
if you dont want to have a separate role i would add the package to the libvirt role instead.

Reply:
we are installing ceph-common to provide udev rules
https://github.com/openstack-k8s-operators/edpm-ansible/blob/main/roles/edpm_libvirt/defaults/main.yml#L52

so nvme-cli would be similar in that regard

Reply:
I'm about to push an update that adds another commit, which will isolate the current iscsid and multipathd roles in their own playbook, to which I'll add (in the other commit) a new nvme role.

I'll invoke this new playbook directly from nova's existing playbook. That will:
- ensure storage stuff is only configured on nova nodes
- ensure storage stuff is always configured wherever nova is deployed
- eliminates the need for creating a new dataplane service